package.yaml

```
@@ -0,0 +1,53 @@
+name:                public-api
+version:             1
```

It might be a good idea to use a real SemVer version number so that we can communicate breaking changes. 
```suggestion
version:             1.0.0
```